app/src/main/java/org/wikipedia/views/SurveyDialog.kt

```diff
@@ -18,15 +27,52 @@ import org.wikipedia.history.HistoryEntry
 import org.wikipedia.settings.Prefs
 import org.wikipedia.util.DimenUtil
 import org.wikipedia.util.FeedbackUtil
+import org.wikipedia.util.UriUtil
 
 object SurveyDialog {
+    // @TODO: update the strings after product confirm them
+    fun showContributionSurveyDialog(
+        activity: AppCompatActivity,
+        @StringRes titleId: Int,
+        @StringRes messageId: Int,
+        @DrawableRes icon: Int? = null,
```

Comment:
I think we will only need one survey verbiage for this experiment, maybe we can either set the title and message directly in the dialog, or set default values for title, message and icon?

Reply:
removed from here and moved to SuggestedEditsTasksFragment